src/actions/addCard.js

```diff
@@ -0,0 +1,13 @@
+import { CardTravelSharp, Store } from "@material-ui/icons";
+import {collection, addDoc} from "firebase/firestore";
+import db from '../Utils/firebaseConfig';
+
+export default async function addCard(card){
+    await addDoc(collection(db,"cards"),card);
```

Comment:
Only define relevant action here. Move this to a new saga.

Reply:
Moved addCard to a new saga